# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/slrn/popup_win.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/menu.c	Wed Mar 13 14:04:23 2002
+++ ./src/menu.c	Wed Apr  3 20:11:15 2002
@@ -915,7 +915,7 @@
 	     tab++;
 	     len += 8;
 	  }
-	if (NULL == (curr->data = slrn_malloc (len, 1, 1)))
+	if (NULL == (curr->data = slrn_malloc (len+1, 1, 1)))
 	  {
 	     free_select_list (root, 1);
 	     return -1;
